example_makefiles/makefile.inc.Mac.brew

```diff
@@ -5,7 +5,7 @@
 # brew install llvm
 CC=/usr/local/opt/llvm/bin/clang++
 CFLAGS=-fPIC -m64 -Wall -g -O3 -msse4 -mpopcnt -fopenmp -Wno-sign-compare -I/usr/local/opt/llvm/include -std=c++11
-LDFLAGS=-g -fPIC -fopenmp -L/usr/local/opt/llvm/lib
+LDFLAGS=-g -fPIC -fopenmp -L/usr/local/opt/llvm/lib -L/usr/local/Cellar/llvm/5.0.0/lib
```

Comment:
Why is this needed?

Reply:
Because homebrew doesn't install libomp to /usr/local/lib